var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -69,6 +69,9 @@ class Perl(Package):  # Perl doesn't use Autotools, it should subclass Package
     variant('cpanm', default=True,
             description='Optionally install cpanm with the core packages.')
 
+    variant('useshrplib', default=False,
```

Comment:
Well, clearly I *can*.  :smirk:

I thought it would be better to stay true to Perl's lingo, but this is an interface between layers and I can see the argument for Spack's view being consistent.

Do other packages use the term `shared` to mean `shared` or do they use it to mean `package-a-word-for-shared` and `package-b-word-for-shared` and ...